checks/search_checks_yaml.go

```diff
@@ -9,4 +9,12 @@ const searchChecksYAML = `
   description: Check disk space errors in Exhibitor logs
   fileTypeName: exhibitor-log
   searchString: No space left on device
+- name: oom-detect
+  description: Detect OOM-killer occurrences
+  fileTypeName: dmesg-t
```

Comment:
The `dmesg-t` file type exists for all kinds of directories: masters, agents, and public agents. If you'd like to check for Java OOM kills only on masters, I think, to avoid looking through all the dmesg files, it would be better to create a new file type specifically for dmesg files on masters, and call it something `dmesg-masters`.

Reply:
I totally agree on this Andrey. Once I get the time I'll dive into the PR some more to add the specific checks on DMESG for masters and agents. thanks for the open discussion!